src/ZPublisher/Converters.py

```diff
@@ -11,6 +11,14 @@
 #
 ##############################################################################
 
+"""Converters
+
+Used by `ZPublisher.HTTPRequest` and `OFS.PropertyManager`.
+
+Converters which work best with binary input have a true `binary` attribute`.
```

Comment:
"work best with" is not very precise. Did you mean to say "returns binary data"?

Reply:
Actually, the input is more important. The converter API does not allow to pass encoding information. Therefore, binary converters must guess the encoding for non binary input (they use the `default_encoding`).

Currently, there is a single (built-in) binary converter (`bytes`). But, in principle, we could have a complete family (`bytes`, `btoken`, `blines`, `btext`, ...).